packages/lesswrong/components/tagging/TagPageActionsMenu.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from 'react';
+import { Components, registerComponent } from '@/lib/vulcan-lib/components';
+import { defineStyles, useStyles } from '@/components/hooks/useStyles';
+import MoreVertIcon from '@material-ui/icons/MoreVert';
+import Menu from '@material-ui/core/Menu';
+import { MAIN_TAB_ID, TagLens } from '@/lib/arbital/useTagLenses';
+import { useTracking } from '@/lib/analyticsEvents';
+import { useMutation, gql, useApolloClient } from '@apollo/client';
+import { useCurrentUser } from '../common/withUser';
+import { userIsAdminOrMod } from '@/lib/vulcan-users';
+import { useMessages } from '../common/withMessages';
+import { captureException } from '@sentry/core';
+
+const styles = defineStyles("TagPageActionsMenu", (theme: ThemeType) => ({
+  tagPageTripleDotMenu: {},
+  icon: {},
+  menu: {},
+  infoCircle: {
+    color: theme.palette.greyAlpha(0.5),
+    marginLeft: 6,
+    "--icon-size": "16px",
+  },
+}))
+
+const TagPageActionsMenuButton = ({tagOrLens, createLens}: {
+  tagOrLens: TagLens|undefined
+  createLens: (() => void)|null,
+}) => {
+  const classes = useStyles(styles);
+  const [anchorEl, setAnchorEl] = useState<any>(null);
+  const [everOpened, setEverOpened] = useState(false);
+  const { captureEvent } = useTracking({eventType: "tagPageMenuClicked", eventProps: {tagOrLensId: tagOrLens?._id, itemType: "tag"}});
+  
+  if (!tagOrLens) {
+    return null;
+  }
+
+  return <>
+    <span
+      className={classes.tagPageTripleDotMenu}
+      onClick={event => {
+        setAnchorEl(event.currentTarget)
+        setEverOpened(true);
+      }}
+    >
+      <MoreVertIcon className={classes.icon}/>
```

Comment:
Prefer `ForumIcon` (we already have `MoreVert` in there; it's called `EllipsisVertical`)